src/ui/public/config/defaults.js

```diff
@@ -109,7 +109,7 @@ define(function (require) {
       'visualization:colorMapping': {
         type: 'json',
         value: JSON.stringify({
-          'Count': '#57c17b'
+          'Count': '#6eadc1'
```

Comment:
Is there a reason to only replace the first seed color? Why not replace them all?